### DISCUSSION TITLE
Adding properties to Markers - first version

### DISCUSSION THREAD

**Discussion by Batuhanbyr** [Arquisoft/lomap_en3b#49] [Show and tell]
I am currently implementing marker properties in my project, and this is the initial version. With this version, users can add a name, review, rating, and privacy status (public or private) to markers. These properties are visible in the info window, providing a quick overview of the marker's details.

This functionality enables users to add more context to markers and distinguish between public and private markers. Future versions may include additional properties and customization options.

**If you have any suggestions or feedback, please let me know in the comments**

https://user-images.githubusercontent.com/95686987/226136391-be7ef8ee-a15c-4562-861a-430296e20e81.mp4

**Comment by Batuhanbyr** [comment]
https://user-images.githubusercontent.com/95686987/226173268-00d92bdd-7c5c-4a93-bd46-fd3feddc6a37.mp4

**Comment by FdezAriasSara** [comment]
This is outstanding :)

**Comment by Batuhanbyr** [comment]
##### Hi!
- a week and that's all I could do 🤡
- can we determine the type of first moment we add the marker? (private, public)? We cannot change its type afterwards. Has this been decided?
https://user-images.githubusercontent.com/95686987/228675786-9647b029-9210-449d-85a6-64865cdc1b9b.mp4

**Reply by FdezAriasSara** [reply]
fist of all thats a really nice job :) 


I don't  understand what you mean when you say we cannot change it afterwards (?) 😟 
. I will  assign this issue to the rest of the team so they can see it too.

**Reply by FdezAriasSara** [reply]
@juanmglzs @Manuhcuartas @RaduSeba

**Reply by juanmglzs** [reply]
It is really nice 😄
I don't know if I'm understanding it... are we taking about what will be the default option when creating a new location (private or public)? 
And I don't understand what does "_We cannot change its type afterwards_" mean. We have talk about changing the privacy level of a location or a review and I think we have said that it will be possible to do so. That I have post a location as private, it can be changed to public at anytime, and vice versa... 
But I don't know if that what your asking.

**Reply by Batuhanbyr** [reply]
Yes i was asking that. Thank you
